src/qutip_qip/circuit.py

```diff
@@ -349,6 +348,18 @@ def add_gate(self, gate, targets=None, controls=None, arg_value=None,
                             control_value=control_value)
                 self.gates.insert(position, gate)
 
+    def add_gates(self, gates):
+        """
+        Add a sequence of gates to the circuit.
```

Comment:
We should perhaps document the order in which the gates are added -- i.e. that this is equivalent to `for g in gates: self.add_gate(g)`.

Reply:
@hodgestar Is there a better way to describe this besides stating `gates are added from a tuple sequentially i.e. from left to right`.

Reply:
I think there is no need to specify it to be a tuple because anything iterable will work here. Maybe we can say `The gates will be applied in positive order, i.e., the first gate in the sequence will be applied first to the state.`